modules/rhythmoneBidAdapter.js

```diff
@@ -1,12 +1,24 @@
 'use strict';
 
+import * as utils from 'src/utils';
```

Comment:
Due to a recent merge (see #3435), we now require any imports for a core or `modules` file (like this adapter file) to use a relative path style.

Can you please update this line here?

Reply:
I have updated the recent merge mentioned. Please validate.
@jsnellbaker if you still see 204 No Content response, please let me know your location. 
Just want to make sure ads are targeted to your location.